src/lib/rest/rest.cpp

```diff
@@ -193,6 +195,23 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
       ciP->uriParamTypes.push_back(val);
     }
   }
+  else if (key == URI_PARAM_Q)
```

Comment:
It is used in rest.cpp and RestService.cpp, this PR contains all its uses, of course